lib/hooks/useGCFormContext.tsx

```diff
@@ -107,11 +109,11 @@ export const GCFormsProvider = ({
   // Note: this only removes the group entry and not the values
   const clearHistoryAfterId = (groupId: string) => _clearHistoryAfterId(groupId, history.current);
 
-  // Note: once localized versions of the section name have been added, update the below to use
-  // something like `getLocalizedProperty("name", locale || "en")`
-  const getGroupTitle = (groupId: string | null) => {
+  const getGroupTitle = (groupId: string | null, language: Language) => {
     if (!groupId) return "";
-    return groups[groupId]?.name || "";
+    const titleLanguageKey = getLocalizedProperty("title", language) as "titleEn" | "titleFr";
+    const title = groups?.[groupId]?.[titleLanguageKey];
+    return title || groups[groupId]?.name || "";
```

Comment:
I was bit on the fence here. If there is no group title, then showing the group name seems like better UX. But then the function is kind of lying :) where it "may" return a group title :)

Reply:
Better not to show as the name won't be a translated string.